cluster/shard_mapper.go

```diff
@@ -230,10 +230,21 @@ func (r *RemoteMapper) NextChunk() (chunk interface{}, err error) {
 			}
 			aggValues = append(aggValues, v)
 		}
-		mo.Values = []*tsdb.MapperValue{&tsdb.MapperValue{
-			Value: aggValues,
-			Tags:  mvj[0].Tags,
-		}}
+
+		if mvj[0].Time != 0 {
```

Comment:
I'm not really sure why it this would make a difference. Either way you end up with a `Time` field equal to 0, right?